hadoop-ozone/ozone-manager/src/main/resources/webapps/ozoneManager/om-overview.html

```diff
@@ -49,6 +45,36 @@ <h2>Status</h2>
     </tbody>
 </table>
 
+<h2>OM Roles (HA)</h2>
+<h4 ng-show="$ctrl.overview.jmx.RatisRoles.length == 1 && $ctrl.overview.jmx.RatisRoles[0].length == 1">Exception: {{$ctrl.overview.jmx.RatisRoles[0]}}</h4>
```

Comment:
I don't think we need the `Expection:` here, we have the necessary information from the `getRatisRoles()` method.
Also if we use `{{$ctrl.overview.jmx.RatisRoles[0][0]}}` we won't have the exception showed like this: `["Exception: Not a Leader"]`, but like this: `Exception: Not a Leader`, right? I think it would be better this way.
```suggestion
<h4 ng-show="$ctrl.overview.jmx.RatisRoles.length == 1 && $ctrl.overview.jmx.RatisRoles[0].length == 1">{{$ctrl.overview.jmx.RatisRoles[0][0]}}</h4>
```